### DISCUSSION TITLE
options for collecting syslog from endpoints

### DISCUSSION THREAD

**Discussion by abesinger** [Security-Onion-Solutions/securityonion#4472] [Unsupported Versions]
I think my question is, with a distributed SO, should I forward endpoint syslog and Windows event logs to a forward node, or directly to logstash on the Manager node? It seems that SO is designed more to forward directly to the Manager.

I like the idea of using a Forward node as an intermediate collector (what I used to call a "branch collector"), giving me the ability to distribute the forwarding load and add some resilience/failover from outages on the central collector (the Manager node), and minimizing disruption to the endpoints.  Basically, I am envisioning something like:

`endpoint (filebeat) --> Forward node (filebeat) --> Manager (logstash)`

However, filebeat doesn't have a "filebeat" input or output. I'd have to use syslog on the endpoint, e.g.

`endpoint (syslog/tcp)) --> Forward node (filebeat) --> Manager (logstash)`

But I don't think that rsyslog is as resilient as filebeat for reliable forwarding. 

So if I want to use filebeat on the endpoint, it seems that I have to send directly to logstash on the Manager node, and not use the Forward node?  E.g.

`endpoint (filebeat)  --> Manager (logstash)`

Am I missing something? I'm not interested in solutions that require installing and supporting Yet Another Software Package on endpoints, I'm looking at either rsyslog or filebeat.

**Comment by weslambert** [comment]
If you are using a Beat, you will need to point it to the manager at this time.

You can point syslog to any box and it should work fine (assuming the FW port is open).

We have definitely considered Beats to sensor communication, however we don't have a supported solution at this time.
